_studio/shared/asc/include/iofunctions.h

```diff
@@ -28,7 +28,7 @@ using namespace ns_asc;
 #pragma warning(disable:4505)
 #endif
 
-static mfxF64 TimeMeasurement(LARGE_INTEGER start, LARGE_INTEGER stop, LARGE_INTEGER frequency) {
+static inline mfxF64 TimeMeasurement(LARGE_INTEGER start, LARGE_INTEGER stop, LARGE_INTEGER frequency) {
```

Comment:
Fix warning "Wunused-function". This is a one line function which use only 1 time in _studio/shared/asc/src/iofunctions.cpp:43.